src/Core/Translation/Storage/Provider/ThemeCatalogueLayersProvider.php

```diff
@@ -137,12 +137,23 @@ public function getDefaultCatalogue(
      *
      * @return MessageCatalogue
      *
-     * @throws TranslationFilesNotFoundException
+     * The **file** translated catalogue for a theme other than classic corresponds
+     * to the core files, overwrite by the user-translated core wordings (if any), overwrite
+     * by the theme files (if any)
      */
     public function getFileTranslatedCatalogue(string $locale): MessageCatalogue
     {
         // load front office catalogue
-        $catalogue = $this->coreFrontProvider->getFileTranslatedCatalogue($locale);
+        $coreCatalogue = $this->coreFrontProvider->getFileTranslatedCatalogue($locale);
+
+        // load core user-translated catalogue
+        $coreUserTranslatedCatalogue = (new UserTranslatedCatalogueFinder(
```

Comment:
Shouldn't `UserTranslatedCatalogueFinder` be injected instead?

Reply:
Or at least use:

```suggestion
        $coreUserTranslatedCatalogue = $this->getUserTranslatedCatalogue($locale);
```

Which has the exact same code (this doesn't mean it can't be injected and used in the method)

Reply:
It's not the **exact** same code, in the `getUserTranslatedCatalogue()` method, we add the theme name, not here 🙃